src/main/java/core/basesyntax/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,71 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.BetDaoImpl;
+import core.basesyntax.dao.UserDao;
+import core.basesyntax.dao.UserDaoImpl;
+import core.basesyntax.model.Bet;
+import core.basesyntax.model.User;
+import java.util.Scanner;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class ConsoleHandler {
+    private static final String NAME_REGEX = "[a-zA-Z]+";
+    private static final String EMAIL_REGEX = "[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\\.[a-zA-Z]{2,6}$";
+    BetDao betDao = new BetDaoImpl();
+    UserDao<User> users = new UserDaoImpl();
```

Comment:
```suggestion
    BetDao betDao = new BetDaoImpl();
    UserDao<User> users = new UserDaoImpl();
```
add please access modifier

Reply:
done